templates/00-default.conf

```diff
@@ -61,14 +61,21 @@ datasources = prometheus
 {{ if (index . "MemcachedServers") }}
 [cache]
 {{ if .MemcachedTLS }}
-backend = dogpile.cache.pymemcache
+backend = oslo_cache.memcache_pool
 memcache_servers={{ .MemcachedServers }}
+memcache_socket_timeout = 0.5
+memcache_pool_connection_get_timeout = 1
+# workaround to force bmemcache driver
+memcache_sasl_enable = true
```

Comment:
The actual parameter name is memcache_sasl_enable**d**

https://github.com/openstack/oslo.cache/blob/9432d7289e322f84ede82838b2647c0186eb9126/oslo_cache/_opts.py#L128

Reply:
thanks for catching this!!

Reply:
Whould we apply a similar patch to the keystone_authtoken configuration? we are configuring memcached and it seems they apply a similar approach:

https://github.com/openstack/keystonemiddleware/blob/86855617f8ed219278dc5e7d0bd93d0ef3cd2d67/keystonemiddleware/auth_token/_cache.py#L80-L83